### ISSUE TITLE
Boost UI improvements

### ISSUE THREAD

**Issue by LexLuthr** [filecoin-project/boost#1824]
The deal list page was designed during the first iteration of the Boost UI. As we progressed over time, it has become clear that some of the items displayed there are not relevant anymore. I propose following changes to the deal list:

1. The deal status/message is "Announcing" once AP() is complete till Indexing and announcing is complete. This can confuse users that deal is stuck as announcing state. Since both sealing stages and indexing are parallel ops, we should display them both.
| State |
| Sealing State (Indexed/Announced) |

2. The size column should use the a different parameter for offline deals.
3. We should display chain deal ID (allocation ID when DDO is live)
4. Display deal UUID in full
5. Removing client ID
6. Display End epoch and expired tag for expired deals

**Comment by LexLuthr** [comment]
@nonsense @ischasny @masih I would love your input here before we start making the changes.

**Comment by masih** [comment]
Thanks for capturing this. They all seem reasonable
No. 1 seems like the most impactful thing to do. My vote would be to concentrate on that one first.

Re "deals getting stuck on Announcing" issue, are we positive that this is purely a UI problem and not a signalling problem between internal boost workflows?

**Comment by LexLuthr** [comment]
I suspect there is something in the deal flow causing that behaviour. The UI just compounds the problem by showing "Announcing" the whole team even when sealing is progressing in back end.

**Comment by LexLuthr** [comment]
I am closing this issue based on https://github.com/filecoin-project/boost/pull/1837
We covered most of the proposed changes and UI looks less confusing now.

**Comment by brendalee** [comment]
- Need for Boost landing page
  - currently shows the list of deals which is the detailed view of the first item on the left-hand side menu
  - ideally, we should have an "Admin Dashboard" like view with a quick summary of info that's most useful to SPs, e.g. Number of clients , data onboarded per clients, expiring deals, etc.
  - The list of what should be on that dashboard needs further brainstorming.
  - Having a placeholder for the dashboard with the summary of what's already presented on the left-hand side bar is a good place to start.
- storage deals page to have a separate column for sealing state (on-chain state) vs lifecycle state. This should reduce the confusion for SPs specifically in the context of on-chain message publication vs indexing announcements to IPNI for example.
- General formatting update on numerical fields to make it easier to search the chain, e.g. deal ID though numerical should not use comma.
- Rendering of UI must remain light-weight. This means careful crafting of DB views, or potential denormalisation of tables.
  - Nothing actionable in this area yet.
- Re priority, focus on low hanging fruits, like deal state separation.
  - Full blown landing page / dashboard needs scoping to judge where it sits in the priorities.

**Comment by brendalee** [comment]
cc @LexLuthr can you make sure the issues that we haven't addressed yet are captured in this issue (or another one, if you are tracking elsewhere)?

**Comment by LexLuthr** [comment]
@brendalee All the improvements agreed upon in the meeting were addressed by #1837 
The landing page is just an idea with no concrete details. It is not a priority and will probably not be in future as well unless requested by the SPs. The amount of work required vs value are very disproportionate.
